modal_proto/api.proto

```diff
@@ -1257,6 +1257,11 @@ message VolumeCommitRequest {
   string volume_id = 1;
 }
 
+message VolumeGetFileRequest {
+  string volume_id = 1;
+  bytes path = 2;
```

Comment:
Why `bytes` as opposed to `string`?

We use `string` pretty much everywhere for paths, including for listing entries in a volume:
```
message SharedVolumeListFilesEntry {
  enum FileType {
    UNSPECIFIED = 0;
    FILE = 1;
    DIRECTORY = 2;
  }
  string path = 1;
  FileType type = 2;
}
```

Reply:
Filepaths in the volume are `bytes` so it seemed incorrect to only accept UTF-8 strings. It would make certain files in a volume inaccessible by the RPC. 

SharedVolume RPCs indeed use `string` and I initially did that, but switched because I thought it was a bug.

Reply:
Fair enough! Makes sense.